src/Sylius/Bundle/ThemeBundle/composer.json

```diff
@@ -36,14 +36,16 @@
         "phpunit/phpunit": "^4.1",
         "symfony/twig-bundle": "^2.7.7"
     },
-    "minimum-stability": "dev",
-    "prefer-stable": true,
```

Comment:
Btw. that Symfony package doesn't have `.travis.yml` too, it's meant to be tested as a part of `symfony/symfony`.